src/bevent.c

```diff
@@ -724,7 +724,8 @@ void ua_event(struct ua *ua, enum ua_event ev, struct call *call,
 		warning("Used deprecated ua_event() for %s. "
 			"Use one of event_xxx_emit() instead!\n",
 			uag_event_str(ev));
-	struct bevent event = {ev, buf, 0, false, { 0 } };
+	struct bevent event = {.ev = ev, .txt = buf, .err = 0, .stop = false,
```

Comment:
Empty values are not needed to initialize so ` {.ev = ev, .txt = buf)` is enough